### ISSUE TITLE
New default data output format breaks scripts that expect simple output

### ISSUE THREAD

**Issue by Selfeer** [ClickHouse/ClickHouse#62042]
With the new ClickHouse 24.3, it seems that there is a new default data output format for queries. 

For example, a simple query in the terminal results in an output with extra symbols and a number around it:

```
root@clickhouse1:/ clickhouse client -q "SELECT 1;"
   ┌─1─┐
1. │ 1 │
   └───┘
```

Previously, the output would just return the expected value:
```
root@clickhouse1:/ clickhouse client -q "SELECT 1;"
1
```

`clickhouse client -q` is used in a lot of different scripts for different purposes, and these scripts expect a simple output like before. 

Given that the default output format was being used for so long, what was the main purpose of changing it now?

**Comment by alexey-milovidov** [comment]
@Selfeer, does it actually break scripts?

**Comment by Selfeer** [comment]
@alexey-milovidov, you are right, it seems that this output format doesn't break regular scripts. For example, if we use the bash script or just pipe the output to `tee` it is a regular simple output.

```                  
root@6c9362b3682b:/# clickhouse client -q "SELECT version()"
   ┌─version()───┐
1. │ 24.3.1.2672 │
   └─────────────┘
```
```
root@6c9362b3682b:/# clickhouse client -q "SELECT version()" | tee
24.3.1.2672
```

But it still breaks any pty scripts that rely on the terminal output. 


There is also one more thing, I followed the ClickHouse [quickstart guide](https://clickhouse.com/docs/en/getting-started/quick-start) from documentation and from the binary I got a `24.4.1.188` version of ClickHouse and for that version it seems that the default output format is back to normal when done with `clickhouse client -q`.

```
user@test:/# clickhouse client -q "SELECT version();"      
24.4.1.188
```

And inside the clickhouse client itself it's still the new format

```
SELECT version()

Query id: 354d0170-98ba-4e76-865b-1c8be5072aab
   ┌─version()──┐
1. │ 24.4.1.188 │
   └────────────┘
```

It's confusing why do we change to a new output format on one version and then change it back on another.
